app/shared/resource-map/MapContainer.js

```diff
@@ -93,4 +102,12 @@ export class UnconnectedResourceMapContainer extends React.Component {
   }
 }
 
-export default connect(selector)(geolocated()(UnconnectedResourceMapContainer));
+function mapDispatchToProps(dispatch) {
```

Comment:
- [ ] This was the old way of giving dispatch to actions in Varaamo. Since this is a new container you can just give the connect the actions object. 
```javascript
const actions = {
  selectUnit,
};

export default connect(selector, actions)(geolocated()(UnconnectedResourceMapContainer));
```

That way the selectUnit will be just a function in the props. A lot nicer to use and mock in tests.